### PR TITLE
fix: kata: workaround passthru net mode when get pod ip

### PR DESCRIPTION
**Pull request by zhuangqh**
Signed-off-by: Ace-Tang <aceapril@126.com>

### Ⅰ. Describe what this PR did
kata: workaround passthru net mode when get pod ip

### Ⅴ. Special notes for reviews
cherry-pick

### CODE REVIEW

File: cri/v1alpha2/cri.go [hunk 1]
```diff
@@ -608,6 +614,10 @@ func (c *CriManager) PodSandboxStatus(ctx context.Context, r *runtime.PodSandbox
 		}
 	}
 
+	if v, exist := annotations[passthruKey]; exist && v == "true" {
+		ip = annotations[passthruIP]
```
Comment (HusterWan) [inline cri/v1alpha2/cri.go]: No need to check the existence of `passthruIP`?

**Comment by allencloud** [comment]
Do we need to fix cri/v1alpha1/cri.go?

This fixes v1alpha2.

File: cri/v1alpha2/cri.go (hunk 1 above)
Comment (rudyfly) [reply cri/v1alpha2/cri.go]: `passthruIP` can be empty string?

**Comment by rudyfly** [comment]
LGTM

**Comment by zhuangqh** [comment]
> Do we need to fix cri/v1alpha1/cri.go?
> 
> This fixes v1alpha2.

v1alpha1 is deprecated. We won't provide the maintenance.

File: cri/v1alpha2/cri.go [hunk 2]
```diff
@@ -74,6 +74,12 @@ const (
 
 	// networkNotReadyReason is the reason reported when network is not ready.
 	networkNotReadyReason = "NetworkPluginNotReady"
+
+	// passthruKey to specify whether a interface is passthru to qemu
+	passthruKey = "io.alibaba.pouch.runv.passthru"
```
Comment (flyer103) [inline cri/v1alpha2/cri.go]: `io.alibaba.pouch.runv.passthru` or `io.alibaba.pouch.vm.passthru`？ @Ace-Tang

Comment (Ace-Tang) [reply cri/v1alpha2/cri.go]: You are right, should be `io.alibaba.pouch.vm.passthru`, could you like help to fix this ? @flyer103

**Pull request merged**